### ISSUE TITLE
Support Easel <-> DiffEq translation

### ISSUE THREAD

**Issue by edhille** [GaloisInc/ASKE-E#57]
A core premise of ASKE-E has been the ability to translate between common representations and Easel. There is a gap still wrt DiffEq that we need to close before the demo

**Comment by yav** [comment]
I don't think this should be a priority, as it is not clear how to do it.   Also, it doesn't seem as important that we translate everything into Easel, but rather that we provide compatible functionality on all models (e.g. simulation).  The benefit of going through Easel in most cases is that it can save work, as we just need to make things work on Easel once, and then reuse this functionality.

When, simulating ODEs, however, we are translating Easel into ODEs, so the other direction seems less useful/important.

**Comment by samcowger** [comment]
As to whether or not this is a priority: from my read of the demo script in #69, we'll need this if we want to compose a model represented as DEQs with one represented as a reaction network, since we'll need for both of them to end up in Core. As to how to do it, @ewdavis and I (and @m10f I think) have discussed a way forward that should work sufficiently well for the cases we care about and/or care to demo, even if not for all cases.
